docs-ref-services/virtualmachines.md

```diff
@@ -33,9 +33,13 @@ pip install azure-mgmt-compute
 
 ### Example
 
-Create a new Linux virtual machine in an existing Azure resource group.
+Create a new Linux virtual machine in an existing Azure resource group with Managed Service Identity(MSI) authentication.
```

Comment:
We should be keep sample without MSI. I was thinking to add a link in the "Samples" section.